### PR TITLE
OAuth Applications

### PR DESCRIPTION
**Pull request by jodosha**
This article documents how to manage OAuth Applications.

### CODE REVIEW

**Comment by jodosha** [comment]
@dnsimple/dnsimple Please review :smile:

**Comment by arnamak** [comment]
LGTM @jodosha 👍

File: content/articles/oauth-applications.markdown [hunk 1]
```diff
@@ -27,11 +27,11 @@ Click on the "Developer applications" tab, then on the "New application" link.
 
 ![New Application Link](/files/new-application-link.png)
 
-Fill the form with "Application Name", "Homepage URL" and "Authorization Callback URL" informations, then click on "Add application" button.
+Fill out the form with the "Application Name", "Homepage URL", and "Authorization Callback URL" information—then click on the "Add application" button.
```
Comment (jodosha) [inline content/articles/oauth-applications.markdown]: Ouch, Oxford comma! :joy_cat: :policeman:

File: content/articles/oauth-applications.markdown [hunk 2]
```diff
@@ -0,0 +1,44 @@
+---
+title: OAuth Applications
+excerpt: Information about the OAuth applications available for DNSimple API.
+categories:
+- API
+---
+
+# OAuth Applications
+
+In order to build a DNSimple addon, you need to create a new OAuth Application.
```
Comment (weppos) [inline content/articles/oauth-applications.markdown]: @jodosha perhaps it is worth to spend one more line explaining what is an app?

File: content/articles/oauth-applications.markdown [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+## Getting to the OAuth Applications page
+
+Log into your account and navigate to your account page by clicking the drop-down in the top right portion of the navbar.
+
+Once you are there, click on the "Applications" link in the sidebar.
```
Comment (weppos) [inline content/articles/oauth-applications.markdown]: We generally wrap text labels within a `<label> Applications </label>`

File: content/articles/oauth-applications.markdown (hunk 2 above)
Comment (jodosha) [inline content/articles/oauth-applications.markdown]: @weppos https://github.com/dnsimple/dnsimple-support/commit/b6b1611431e506ba4ac0b5e8ad4902a0dddb00a2 😉

**Pull request merged**